packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1993,6 +2013,9 @@ class _TabsDefaultsM3 extends TabBarTheme {
   late final ColorScheme _colors = Theme.of(context).colorScheme;
   late final TextTheme _textTheme = Theme.of(context).textTheme;
 
+  @override
+  double? get dividerHeight => 1.0;
```

Comment:
Just for follow up after https://github.com/flutter/flutter/pull/122756, I assume this will need to be updated the for M3 defaults for primary & secondary.

Reply:
Yes, I will update this PR immediately once https://github.com/flutter/flutter/pull/122756  land. As tthere will be conflict in the defaults text and as you pointed out, I will add dividerHeight in secondary defaults as well.

Reply:
- Resolved conflicts,
- updated default tests
- modified an existing duplicate test for the secondary tab bar golden
- Updated underline indicator to shift so I can align with the divider.